### PR TITLE
How to use GitHub’s search feature to search the wiki 

### PR DESCRIPTION
**Pull request by MohitGupta121**
How to use GitHub’s search feature to search the wiki
- With this contributor easily search and navigate to desire wiki page.

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@gp201 PTAL, Thanks.

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, Thanks.

File: Contributing-code-to-Oppia.md [hunk 1]
```diff
@@ -18,6 +19,10 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
+
+<img width="1074" alt="Screenshot wiki preview" src="https://github.com/MohitGupta121/oppia-android/assets/76530270/866c6796-3c50-41fe-992a-778728401c8f">
```
Comment (gp201) [inline Contributing-code-to-Oppia.md]: This seems to be using the old UI. Can you update it with the latest github UI?

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @MohitGupta121 ! I left some comments. PTAL

File: Contributing-code-to-Oppia.md (hunk 1 above)
Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: But Praneeth, I using the same UI that is in the video. Which new UI you talking? 
Can you please brief if possible?

Comment (gp201) [reply Contributing-code-to-Oppia.md]: @MohitGupta121 Sorry about that. Looks like I was using a feature preview, so my UI was a bit different.

**Comment by gp201** [comment]
@MohitGupta121, to future-proof it, can you add a set of steps to the top?

**Comment by MohitGupta121** [comment]
@gp201 set of setup means written guided steps for wiki search right?

File: Contributing-code-to-Oppia.md (hunk 1 above)
Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: @gp201 okay sure, no problem. Thanks!

**Comment by gp201** [comment]
@MohitGupta121 Yes

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, updated the wiki gudie.

File: Contributing-code-to-Oppia.md [hunk 2]
```diff
@@ -18,6 +19,13 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
```
Comment (gp201) [inline Contributing-code-to-Oppia.md]: Since the title is "How to find information on this wiki" do you want to include some info about the sidebar also?

**Review by gp201 — CHANGES_REQUESTED** [review]

File: Contributing-code-to-Oppia.md (hunk 2 above)
Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: Which sidebar? Sidebar that shown after search on github or sidebar that our wiki have?

Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: Done.

**Comment by MohitGupta121** [comment]
@gp201 PTAL, I updated the requested changes. Thanks!

File: Contributing-code-to-Oppia.md [hunk 3]
```diff
@@ -18,6 +19,19 @@ If you run into any problems along the way, we're here to help! Check out our [[
...
+3. GitHub provides some advanced search filters that you can use to refine your search further. You can filter results by the file type, code language, author, or specific locations within the repository. To use these filters, click on the "Filters" button next to the search bar and select the desired options.
+4. Press Enter or click on the search button to initiate the search.
+5. GitHub will display the search results based on your query. The results will show matching files, code snippets, issues, pull requests, commits, discussions, wikis and other relevant information which you can see in left side after search.
+6. You can click on any search result to view more details or explore the code/files associated with it.
+
+Additionally, GitHub also provides an advanced search syntax that allows you to construct more complex queries. You can find more information about this syntax in the [GitHub documentation](https://docs.github.com/en/github-ae@latest/search-github/searching-on-github/searching-code).
+
+<img width="1074" alt="Screenshot wiki preview" src="https://github.com/MohitGupta121/oppia-android/assets/76530270/866c6796-3c50-41fe-992a-778728401c8f">
```
Comment (gp201) [inline Contributing-code-to-Oppia.md]: Can you make this a video? Also, can you use the white mode for this? The gif and the dark mode seem to make the gif hard to see on some screens. In the gif make sure you go to the "Code" tab, "issues" tab, PR "tab" and the "wiki" tab

File: Contributing-code-to-Oppia.md [hunk 4]
```diff
@@ -18,6 +19,19 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
```
Comment (gp201) [inline Contributing-code-to-Oppia.md]: Rename this title to something like "How to find information in the repo and the wiki" or "how to search for info on the repo".

**Review by gp201 — CHANGES_REQUESTED** [review]
@MohitGupta121 Nice work! Just a few changes to the title and the video.

File: Contributing-code-to-Oppia.md (hunk 3 above)
Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: Done.

File: Contributing-code-to-Oppia.md (hunk 4 above)
Comment (MohitGupta121) [reply Contributing-code-to-Oppia.md]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
@gp201 PTAL, Thanks.

**Review by gp201 — APPROVED** [review]
LGTM

**Comment by MohitGupta121** [comment]
@gp201 so now the PR is ready to merge?

**Pull request merged**